core/store/models/cbor_test.go

```diff
@@ -120,11 +128,105 @@ func Test_autoAddMapDelimiters(t *testing.T) {
 	}
 }
 
-func jsonMustUnmarshal(in string) JSON {
+func jsonMustUnmarshal(t *testing.T, in string) JSON {
 	var j JSON
 	err := json.Unmarshal([]byte(in), &j)
-	if err != nil {
-		log.Panicf("Failed to unmarshal '%s'", in)
-	}
+	require.NoError(t, err)
 	return j
 }
+
+func TestCoerceInterfaceMapToStringMap(t *testing.T) {
```

Comment:
this might be a little cleaner as `TestCoerceInterfaceMapToStringMap_Happy` and `TestCoerceInterfaceMapToStringMap_Error`, since only "want" or "wantError" are relevant in the test cases.